integration/helpers/base_test.py

```diff
@@ -567,9 +567,9 @@ def verify_options_request(self, url, expected_status_code, headers=None):
             )
         return response
 
-    def verify_post_request(self, url: str, body_obj, expected_status_code: int):
+    def verify_post_request(self, url: str, body_obj, expected_status_code: int, headers=None):
```

Comment:
I think we need to pass the headers [here](https://github.com/aws/serverless-application-model/blob/20643062084d3a1d3b3c7e626687bccaf9650a64/integration/single/test_basic_api.py#L128) to the calling method. If not it will be empty i think

Reply:
My understanding is [this part](https://github.com/aws/serverless-application-model/blob/f19a827ef4444e1a4e611b05d7f664b8f756daf7/integration/helpers/base_test.py#L656) of the code (whose class is defined [here](https://github.com/aws/serverless-application-model/blob/f19a827ef4444e1a4e611b05d7f664b8f756daf7/integration/helpers/request_utils.py#L13)) will match the headers outputted by the response to the headers defined in amazon headers method. For example if there is an error the `x-amzn-errortype` header will be set. Do you know of any other headers we should be passing in here?

Reply:
Thanks @aaythapa. Seems like the headers passed to the method are request headers which will be used while making the api call and not for logging. The headers we log are from the api response and these headers which we already defined [here](https://github.com/aws/serverless-application-model/blob/f19a827ef4444e1a4e611b05d7f664b8f756daf7/integration/helpers/request_utils.py#L4-L10) are good. 
May be it is good to rename `headers` variable to `requestHeaders` so it will be more clear. Its nonblocking and good to update in next PR. 

LGTM. Thanks @aaythapa for the fix.